charts/actions-runner-controller/values.yaml

```diff
@@ -21,6 +21,9 @@ syncPeriod: 10m
 authSecret:
   create: false
   name: "controller-manager"
+  ## Set specific secret annotations
+  # annotations: |
```

Comment:
I think you must omit `|` otherwise the whole `annotations` becomes a YAML string and it doesn't work correctly when piped to `toYaml`, right?